packages/design-picker/src/types.ts

```diff
@@ -13,36 +14,46 @@ export interface Category {
 	name: string;
 }
 
+export interface DesignRecipe {
+	theme?: string;
+	patternIds?: number[];
+}
+
 export type DesignFeatures = 'anchorfm' | 'difm-lite-default'; // For additional features, = 'anchorfm' | 'feature2' | 'feature3'
 
 export interface Design {
-	categories: Array< Category >;
-	fonts?: FontPair;
-	is_alpha?: boolean;
-	is_fse?: boolean;
+	slug: string;
+	title: string;
+	recipe?: DesignRecipe;
 	is_premium: boolean;
+	categories: Category[];
+	features: DesignFeatures[];
+	is_featured_picks?: boolean; // Whether this design will be featured in the sidebar. Example: Blank Canvas
+	showFirst?: boolean; // Whether this design will appear at the top, regardless of category
+	preview?: 'static';
+
+	/** @deprecated used for Gutenboarding (/new flow) */
 	stylesheet?: string;
-	slug: string;
+	/** @deprecated used for Gutenboarding (/new flow) */
```

Comment:
Yeah, but I think after Gutenboarding is gone, the value is "meaningless" as it will always be the same as theme, right? (That's why I also mark this as deprecated.)